nmoscommon/auth/nmos_auth.py

```diff
@@ -60,20 +58,40 @@ def __init__(self, condition=OAUTH_MODE, claimsOptions=IS_XX_CLAIMS):
         self.claimsOptions = claimsOptions
```

Comment:
I can't comment on the specific line in this PR, but just thinking about it, do we end up with an mDNS bridge client instance for every API path which uses the RequiresAuth decorator? If so is there any way we can share a single instance between them? Similar for the rest of the class, does this result in requests for server metadata and keys each time an API path is hit, or is it cached and shared? If cached, is there a procedure for refreshing the keys when they approach expiry?

Reply:
Sharing a single instance of the mDDNSClient should be fine. WRT the keys, the public key is cached _per instance_ of RequiresAuth (so for each endpoint) but not for all instances within a given API. Might this be resolved with global vars? Might this mean that _every_ API running on a given device might then be using the same keys? That may well not be a problem

Reply:
I don't think that is a problem. Using multiple auth servers within a single NMOS deployment isn't specified so there's no need for us to handle it here.